frontend/src/tests/wage-chunk-tests.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */
```

Comment:
Why is eslint disabled?

Reply:
it's complaining about the 
```js
export function wageChunksTests({ apiGET, apiPOST })
```
we need to either use eslint disabled, or not including the passed in arguments for all the tests only have `todo`